### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by myleftfoot**
small fixes to make the project work better with linux/macos

- added missing qs package
- changed fs-extra fs
- removed --exec from nodemon command

### CODE REVIEW

**Comment by jakebrown58** [comment]
Do we need the package-lock.json? 
I suspect we should add that to the .gitignore.

Otherwise, looks good, (since we worked through it over slack).

**Comment by myleftfoot** [comment]
npm install
npm notice created a lockfile as package-lock.json. You should commit this file.
don’t know why they add that stupid -lock file…
but it looks like they want us to commit it.

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "nib": "^1.1.0",
     "node-skytap": "^0.7.0",
     "q": "^1.4.1",
+    "qs": "^6.5.1",
```
Comment (adampaulberg) [inline package.json]: What uses this that it needed added?

Comment (jakebrown58) [reply package.json]: Swarm.   when I run npm install on windows, it is included in the node_modules, but when he ran it from a linux box, it wasn't pulled down.

**Comment by jakebrown58** [comment]
Re: package-lock-json files.

https://stackoverflow.com/questions/44552348/should-i-commit-yarn-lock-and-package-lock-json-files

looks like it's recommended to commit it.

File: package.json (hunk 1 above)
Comment (adampaulberg) [reply package.json]: 👍

Comment (jakebrown58) [reply package.json]: basically, it was always needed, but it used to be implicitly pulled in.

Comment (myleftfoot) [reply package.json]: the client requires this.. otherwise, I was getting component 'qs" not found

**Pull request merged**